course_content/iris_course/solutions/iris_exercise_4.1b

```diff
@@ -0,0 +1,11 @@
+
+# Iris course sample answer 4.1b
+# make a colour blockplot of "airtemps"
+
+try:
+    qplt.pcolormesh(airtemps)
+    plt.show()
+except Exception as err:
+    print('Error occurred:\n  "{}"'.format(err))
+
+print("NOTE: the 'qplt.pcolormesh' method requires a 2d cube.")
```

Comment:
The suggested activity tells the user to do a `qplt.contourf` but this answer uses pcolormesh.
As pcolormesh gets introduced later, I think it makes sense to use contourf.